Packs/Druva/Integrations/Druva/Druva_test.py

```diff
@@ -1,3 +1,75 @@
+        verify=False,
+        headers={
+            'Authentication': 'Bearer some_api_key'
+        }
+    )
+    with pytest.raises(RuntimeError) as ve:
+        test_module(client)
+    assert str(ve.value)=='Internal Error'
+
+def test_Execute_command(requests_mock):
+    from Druva import Client,Execute_command
+    client = Client(
+        base_url='https://apis.druva.com/',
+        verify=False,
+        headers={
+            'Authentication': 'Bearer some_api_key'
+        }
+    )
+    
+    with open("test_data/test-data.json") as file:
+        data=json.load(file)
+        commands=["test-module","druva-list-quarantine-ranges","druva-find-device","druva-find-user","druva-find-userDevice","druva-find-sharePointSites","druva-find-sharedDrives",
+                  "druva-quarantine-resource","druva-delete-quarantine-range","druva-view-quarantine-range","druva-update-quarantine-range","druva-list-quarantine-snapshots","druva-delete-quarantined-snapshot",
+                  "druva-endpoint-check-restore-status","druva-endpoint-search-file-hash"]
+        for cur_command in commands:
+            req_type=data[cur_command]["request-type"]
+            if cur_command=="druva-quarantine-resource":
+                mock_response_post=data[cur_command]["mock-responce"]
+                mock_response_get=data["druva-list-quarantine-ranges"]["mock-responce"]
+                requests_mock.post(data[cur_command]["url-suffix"],status_code=200,json=mock_response_post)
+                requests_mock.get(data["druva-list-quarantine-ranges"]["url-suffix"],status_code=200,json=mock_response_get)
+                Execute_command(cur_command,client)
+            elif cur_command=="druva-update-quarantine-range":
+                mock_response_post=data[cur_command]["mock-responce"]
+                mock_response_get=data["druva-list-quarantine-ranges"]["mock-responce"]
+                requests_mock.put(data[cur_command]["url-suffix"],status_code=200,json=mock_response_post)
+                requests_mock.get(data["druva-list-quarantine-ranges"]["url-suffix"],status_code=200,json=mock_response_get)
+                Execute_command(cur_command,client)
+            elif req_type=="get":
+                mock_response=data[cur_command]["mock-responce"]
+                requests_mock.get(data[cur_command]["url-suffix"],status_code=200,json=mock_response)
+                Execute_command(cur_command,client)
+            elif req_type=="delete":
+                mock_response=data[cur_command]["mock-responce"]
+                requests_mock.delete(data[cur_command]["url-suffix"],status_code=200,json=mock_response)
+                Execute_command(cur_command,client)
+            elif req_type=="post":
+                mock_response=data[cur_command]["mock-responce"]
+                requests_mock.delete(data[cur_command]["url-suffix"],status_code=200,json=mock_response)
+                Execute_command(cur_command,client)
+        
```

Comment:
it is recommended to make some assertions at the end of unit tests.
for this, you can do something like:
```suggestion
        outputs = demisto.results.call_args[0][0]['Contents']
        assert something_about_outputs
```
if you want to do this, you should run at the beginning of the method:
```
mocker.patch.object(demisto, 'results')
```

Reply:
Hi,dantavori
What's problem to have separate Execute_command.

Reply:
I explain in the first comment: https://github.com/demisto/content/pull/26060#discussion_r1175542485
It's not a problem, but if you insist on it, you must bump a version for the pack and add a release note.

Reply:
Bumped the version by note as Internal code improvements.